.circleci/config.yml

```diff
@@ -156,12 +193,16 @@ jobs:
 
 workflows:
   version: 2
+  validate-pr:
+    when: << pipeline.parameters.run-auto-workflows >>
```

Comment:
I had to add this because of a bug? in circle config validation. Without `validate-pr` every workflow execution is disabled unless a pipeline parameter is provided. For this repo that is correct because it only runs 
scheduled workflows but now that we filter pipelines we need to have at least one workflow that runs (until circle fix this?)

See: https://github.com/CircleCI-Public/circleci-cli/issues/577

Without `validate-pr` ⬇️ 
<img width="879" alt="Screen Shot 2022-01-17 at 18 20 21" src="https://user-images.githubusercontent.com/6853656/149838845-698463c1-5ff3-40f4-86b4-8f462143979a.png">